Tests/BugsnagSinkTests.m

```diff
@@ -20,7 +20,7 @@ @interface BugsnagSinkTests : XCTestCase
 @end
 
 @interface BugsnagSink ()
-- (NSDictionary *)getBodyFromReports:(NSArray *)reports;
+- (NSDictionary *)getBodyFromReports:(NSArray *)events;
```

Comment:
This method is only mentioned here and in the test, a total of 6 times, AFAICT.  I'd be happy for it to be renamed to `getBodyFromEvents:` as part of this change; I think it's clear enough housekeeping.

Reply:
Agreed - renamed this method.